x-pack/solutions/observability/plugins/observability/public/pages/alert_details/components/related_dashboards/dashboard_tile.tsx

```diff
@@ -48,24 +49,15 @@ export function DashboardTile({
     <>
       <EuiFlexGroup gutterSize="xs" responsive={false} key={dashboard.id}>
         <EuiFlexItem key={dashboard.id}>
-          <EuiText size="s">
-            <a
-              href="#"
-              onClick={async (e) => {
-                e.preventDefault();
-                if (dashboardLocator) {
-                  const url = await dashboardLocator.getUrl({
-                    dashboardId: dashboard.id,
-                  });
-                  window.open(url, '_blank');
-                } else {
-                  console.error('Dashboard locator is not available');
-                }
-              }}
-            >
-              {dashboard.title}
-            </a>
-          </EuiText>
+          <EuiLink
+            data-test-subj="o11yDashboardTileLink"
+            href={dashboardLocator?.getRedirectUrl({
+              dashboardId: dashboard.id,
```

Comment:
Can we also pass the time range for the alert here? The dashboard locator supports adding a time range.